src/main/java/frc/robot/constants/Constants.java

```diff
@@ -19,8 +19,18 @@ public final class Constants {
   public static final double UP_ADJUST = 0.5;
   public static final double DOWN_ADJUST = 0.25;
 
-  public static final Translation2d BLUE_ORBIT_POSE = new Translation2d(0.5, 5.5);
-  public static final Translation2d RED_ORBIT_POSE = new Translation2d(16, 5.5);
+  // TODO should revise and precisely measure these values, they are estimates
+  public static final Translation2d BLUE_SPEAKER_POSE = new Translation2d(0, 5.5);
+  public static final Translation2d RED_SPEAKER_POSE = new Translation2d(16.5, 5.5);
+  public static final double SPEAKER_HEIGHT = 2.1;
```

Comment:
It'd be nice if we used the Units library just so it was clear what this value represented